src/shapes.tsx

```diff
@@ -1,69 +1,106 @@
+//
+// All icons are imported from https://feathericons.com/ Made by @colebemis
+// github: https://github.com/feathericons/feather
+// Feather is a collection of simply beautiful open source icons.
+// Each icon is designed on a 24x24 grid with an emphasis on simplicity, consistency, and flexibility.
+// Icons are under the license https://github.com/feathericons/feather/blob/master/LICENSE
+// Small adjusments and additions by @tomfejer
+//
 import React from "react";
 import oc from "open-color";
 
-// We inline font-awesome icons in order to save on js size rather than including the font awesome react library
+// We inline icons in order to save on js size rather than including the font awesome react library
 export const SHAPES = [
   {
     icon: (
-      // fa-mouse-pointer
-      <svg viewBox="0 0 320 512" className="">
-        <path d="M302.189 329.126H196.105l55.831 135.993c3.889 9.428-.555 19.999-9.444 23.999l-49.165 21.427c-9.165 4-19.443-.571-23.332-9.714l-53.053-129.136-86.664 89.138C18.729 472.71 0 463.554 0 447.977V18.299C0 1.899 19.921-6.096 30.277 5.443l284.412 292.542c11.472 11.179 3.007 31.141-12.5 31.141z" />
+      <svg
+        width="24"
+        height="24"
+        viewBox="0 0 24 24"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M3 11L22 2L13 21L11 13L3 11Z"
+          stroke={oc.black}
+          stroke-width="2"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+        />
```

Comment:
This one feels better.. https://github.com/feathericons/feather/blob/master/icons/mouse-pointer.svg